app/models/abstract_feature.rb

```diff
@@ -134,37 +173,64 @@ def self.geojson(lowres: false, precision: 6, properties: true, srid: 4326, cent
       column = 'geog'
     end
 
-    properties_sql = <<~SQL if properties
-      , 'properties', hstore_to_json(metadata)
+    properties_sql = []
+
+    if include_record_identifiers
+      properties_sql << "hstore(ARRAY['feature_name', name::varchar, 'feature_id', id::varchar, 'spatial_model_type', spatial_model_type::varchar, 'spatial_model_id', spatial_model_id::varchar])"
+    end
+
+    if properties
+      properties_sql << "metadata"
+      properties_sql << <<~SQL
+        hstore(ARRAY['feature_area', area::varchar])
+      SQL
+    end
+
+    if properties.is_a?(Hash)
+      properties_sql << <<~SQL
+        hstore(ARRAY[#{properties.flatten.map {|e| "'#{e.to_s}'" }.join(',')}])
+      SQL
+    end
+
+    properties_sql = <<~SQL if properties_sql.present?
+      , 'properties', hstore_to_json(#{properties_sql.join(' || ')})
     SQL
 
     sql = <<~SQL
+      json_agg(
+        json_build_object(
+          'type', 'Feature',
+          'geometry', ST_AsGeoJSON(#{column}, #{precision})::json
+          #{properties_sql}
+        )
+      )
+    SQL
+
+    sql = <<~SQL unless features_only
       json_build_object(
         'type', 'FeatureCollection',
-        'features', json_agg(
-          json_build_object(
-            'type', 'Feature',
-            'geometry', ST_AsGeoJSON(#{column}, #{precision})::json
-            #{properties_sql}
-          )
-        )
+        'features', #{sql}
       )
     SQL
     SpatialFeatures::Utils.select_db_value(all.select(sql))
   end
 
-  def feature_bounds
-    slice(:north, :east, :south, :west)
+  def self.bounds
+    values = pluck('MAX(north) AS north, MAX(east) AS east, MIN(south) AS south, MIN(west) AS west').first
```

Comment:
```suggestion
    values = pick('MAX(north) AS north, MAX(east) AS east, MIN(south) AS south, MIN(west) AS west')
```

Reply:
Pick is rails 6, so if we're sure we don't need older compatibility then sure. Thoughts?